hazelcast/src/main/java/com/hazelcast/core/HazelcastInstance.java

```diff
@@ -452,6 +453,13 @@
      */
     @Nonnull CPSubsystem getCPSubsystem();
 
+    /**
+     * Returns SQL service.
+     *
+     * @return SQL service.
```

Comment:
I agree that these comments do not add any value. At the same time, JavaDoc without body or return tag may look aesthetically inconsistent in both IDE, or when converted to HTML. E.g. JDK chooses the approach with repetition. When you look at missing part of JavaDoc the question is whether this is a bug or an intentional omission. With explicit docs it is clear - there is just no more useful information to add.

Therefore I'd prefer to keep it as is, given that we do not have any hard project-wide rules for one or the other approach, and both formatting options could be found in different parts of the API.

Reply:
given that this is the entry point to the API, I think it warrants more than a trivial comment? Perhaps a sentence or two about it wouldn't hurt.

Reply:
Rendered in IntelliJ, how _aesthetically consistent_ is this?
![image](https://user-images.githubusercontent.com/9109877/86051989-95b76780-ba56-11ea-92fe-a9471b8c4360.png)

Maybe we could replace the `@return` with `@see SqlService`. That's also duplicate, one can click on the return type, but it suggests that there's something interesting and related in there.

Reply:
Also, should be marked with `@NonNull`.

Reply:
We do not go into details in `HazelcastInstance`, because extended JavaDocs are provided in the returned objects. I added a reference to `SqlService` in the method's JavaDoc.